src/modules/Home/components/Home/Home.tsx

```diff
@@ -0,0 +1,324 @@
+import cn from 'classnames';
+import React from 'react';
+
+import boxSVG from 'assets/images/box.svg';
+import ChipIcon from 'assets/images/ChipIcon';
+import worldSVG from 'assets/images/continents.svg';
+import devicesSVG from 'assets/images/devices.svg';
+import discountSVG from 'assets/images/discount.svg';
+import githubSVG from 'assets/images/github.svg';
+import LogoIcon from 'assets/images/LogoIcon';
+import octocatSVG from 'assets/images/octocat.svg';
+import openedBookSVG from 'assets/images/openedBook.svg';
+import shieldSVG from 'assets/images/shield.svg';
+import userSVG from 'assets/images/user.svg';
+import Flex from 'common/components/Flex/Flex';
```

Comment:
add space

Reply:
Fixed, but my i cant tell why my space do not shown here